README.rst

```diff
@@ -1606,6 +1627,30 @@ This setting allows to change the logo which is displayed in HTML version of the
 **Note**: Provide a URL which points to the logo on your own web server. Ensure that the URL provided is
 publicly accessible from the internet. Otherwise, the logo may not be displayed in the email.
 
+``OPENWISP_CELERY_SOFT_TIME_LIMIT``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
++---------+---------------------+
+| type    | ``int``             |
++---------+---------------------+
+| default | ``10`` (in seconds) |
++---------+---------------------+
+
+Sets the soft time limit for celery tasks using
+`OpenwispCeleryTask <#openwisp_utilstasksopenwispcelerytask>`_.
+
+``OPENWISP_CELERY_HARD_TIME_LIMIT``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
++---------+---------------------+
+| type    | ``int``             |
++---------+---------------------+
+| default | ``30`` (in seconds) |
```

Comment:
wouldn't these defaults prevent using this class in many tasks we have now scattered around OW?

I'd be more conservative and use 30 seconds and 60 seconds.

Reply:
I think the hard time limit could even default to something higher, see https://github.com/openwisp/openwisp-notifications/pull/224#discussion_r807215214